extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -190,6 +190,13 @@ function createDicomWebApi(dicomWebConfig, servicesManager) {
           params
         );
       },
+      /**
+       * Provide direct access to the dicom web client for certain use cases
+       * where the dicom web client is used by an external library such as the
+       * microscopy viewer.
+       */
+      getWadoDicomWebClient: () => wadoDicomWebClient,
```

Comment:
getDicomWebClientInstance?

Reply:
I added commentary:

Reply:
The issue is that it isn't necessarily a full dicom web client instance, but is specific to the WADO part (retrieve) as some back ends separate out the various services for different reasons.